### PR TITLE
AC-1659 - (UI) Migrate guides from Pendo to Appcues

### PR DESCRIPTION
**Pull request by shrirupa**
AC-1659 - (UI) Migrate guides from Pendo to Appcues

### What Changed
 - Moved following guides from Pendo to Appcues
       - Templates - Onboarding
       - Summary Reports - Onboarding
       - Events Onboarding
       - Sending Domains - Onboarding
       - Cancellation Survey
       - Hibana Feedback Survey

### How To Test
 - Check if all the guides work correctly and look good
 - All guides are here - https://studio.appcues.com/flows

### To Do
- [ ] Address feedback
~- [ ] Do we need to have seperate guides for Hibana ? Since the new dashboard is in progress we can have new guides there.~
- [ ] **When deploying to staging and production remember to update the domain in Appcues.**

### CODE REVIEW

File: src/pages/reports/messageEvents/MessageEventsPage.js [hunk 1]
```diff
@@ -216,6 +216,12 @@ const MessageEventsPage = props => {
   const [state] = useHibana();
   const { isHibanaEnabled } = state;
 
+  useEffect(() => {
+    if (props.location?.state?.triggerGuide) {
```
Comment (nicklemmon) [inline src/pages/reports/messageEvents/MessageEventsPage.js:220]: This is the best new JS feature in years.

File: src/pages/reports/messageEvents/MessageEventsPage.js [hunk 2]
```diff
@@ -216,6 +216,12 @@ const MessageEventsPage = props => {
...
+      window.Appcues.show(GUIDE_IDS.CHECKOUT_EVENTS);
```
Comment (nicklemmon) [inline src/pages/reports/messageEvents/MessageEventsPage.js:221]: should we be checking that `window.Appcues` exists? I think ad blockers might interfere?

File: src/pages/reports/messageEvents/MessageEventsPage.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable max-lines */
```
Comment (nicklemmon) [inline src/pages/reports/messageEvents/MessageEventsPage.js:1]: Don't think we enforce this rule anymore - could we remove this?

File: src/pages/reports/messageEvents/MessageEventsPage.js [hunk 4]
```diff
@@ -216,6 +216,12 @@ const MessageEventsPage = props => {
   const [state] = useHibana();
   const { isHibanaEnabled } = state;
 
+  useEffect(() => {
```
Comment (nicklemmon) [inline src/pages/reports/messageEvents/MessageEventsPage.js:219]: Do we have any automated testing to verify this?

File: src/pages/reports/summary/SummaryPage.js [hunk 5]
```diff
@@ -22,6 +23,10 @@ export class SummaryReportPage extends Component {
     if (prevProps.reportOptions !== this.props.reportOptions) {
       this.props.refreshSummaryReport(this.props.reportOptions);
     }
+
+    if (prevProps.location?.state?.triggerGuide) {
+      window.Appcues.show(GUIDE_IDS.EXPLORE_ANALYTICS);
+    }
```
Comment (nicklemmon) [inline src/pages/reports/summary/SummaryPage.js:29]: Any testing for this?

File: src/pages/sendingDomains/EditPage.js [hunk 6]
```diff
@@ -72,6 +72,12 @@ export class EditPage extends Component {
     this.props.clearSendingDomain();
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.location?.state?.triggerGuide) {
+      window.Appcues.show(GUIDE_IDS.VERIFY_SENDING_DOMAIN);
+    }
+  }
```
Comment (nicklemmon) [inline src/pages/sendingDomains/EditPage.js:79]: Any testing for this?

**Review by nicklemmon — DISMISSED** [review]
@shrirupa Left a few comments - I think mainly just a few gaps in testing that you should look at?

**Review by gfriedmansp — APPROVED** [review]
I think everything looks good now 👍

**Pull request merged**